Resources/Locale/en-US/nyanotrasen/abilities/psionic.ftl

```diff
@@ -63,11 +66,14 @@ action-name-noospheric-zap = Noospheric Zap
 action-description-noospheric-zap = Shocks the conciousness of the target and leaves them stunned and stuttering.
 
 action-name-pyrokinesis = Pyrokinesis
-action-description-pyrokinesis = Light a flammable target on fire.
-pyrokinesis-power-used = A wisp of flame engulfs {THE($target)}, igniting {OBJECT($target)}!
+action-description-pyrokinesis = Hurl a small gateway to the plane of Gehenna at your target.
```

Comment:
a gateway to *where*?

Reply:
Gehenna, a place in Jewish mythology that is analogous to the common depictions of Hell. This esoteric description is given intentionally as a way of tying it to the planned addition of the Sheol-PLC Gehennothermic Engine, with the assumption that the source of fire created by Pyromancers, Ifrits, the SPGE, and Fire Anomalies, are all the same location that we as humans would describe, but is not necessarily literally, Hell.